src/appforms/src/core/020-utils02webcam.js

```diff
@@ -24,10 +24,18 @@ appForm.utils = function (module) {
     checkEnv();
     _html5Camera(params, cb);
   }
+
   function cancelHtml5Camera() {
-    if (localMediaStream) {
-      localMediaStream.stop();
-      localMediaStream = null;
+    if (localMediaStream){
+      if (localMediaStream.stop) {
+        localMediaStream.stop();
+        localMediaStream = null;
+      } else{
+        var tracks = localMediaStream.getTracks();
```

Comment:
no, I mean `tracks` could be undefined? you are using `tracks.length` in the next line. But what if tracks is undefined?